Cargo.toml

```diff
@@ -28,6 +28,8 @@ serde = { version = "1.0", features = ["derive"] }
 serde-value = "0.7"
 tokio = { version = "1.21", default-features = false, features = ["macros", "rt"] }
 tokio-test = "0.4"
+anyhow = "1"
```

Comment:
I think we can do without that dependency. It does not give any special advantages in our example

Reply:
I'm not sure what error to return from main().  If I change to:
```
fn main() -> Result<(), quick_xml::Error> {
```

I get:
```
  --> examples/read_nodes_serde.rs:70:39
   |
64 | fn main() -> Result<(), quick_xml::Error> {
   |              ---------------------------- expected `quick_xml::Error` because of this
...
70 |     let config: Config = from_str(XML)?;
   |                                       ^ the trait `From<DeError>` is not implemented for `quick_xml::Error`
   |
```

How do you recommend writing this example to avoid this compile error?